src/pages/kb/user-guide/alerts/setting-up-an-alert.md

```diff
@@ -14,43 +15,112 @@ title: Setting Up An Alert
 slug: setting-up-an-alert
 order: 1
 ---
-In the Alerts view, you'll see all existing alerts.
+Alerts provide a convenient way for Redash to notify you when your scheduled queries match some arbitrary criteria. They are often used to monitor system statuses, financial goals, and unusual account activity. Through integrations with Zapier and IFTTT they can also trigger business processes like user on-boarding work flows or support tickets.
+
+You can see all of your existing alerts by clicking `Alerts` on the main navigation menu. This will show you all the alerts in your organization by name, user, current state, and created date.
 
 ![](/assets/images/docs/gitbook/alerts.png)
 
-To set up a new alert you'll need to follow these steps:
+# Usage
+
+Create a new Alert by clicking the blue `Create` button in the navigation menu and selecting `Alert`.
+
+![](/assets/images/docs/gitbook/create-alert.png)
+
+Find the query that you would like to monitor using the search bar. Keep in mind that Alerts do not work with queries that use parameters.
+
+![](/assets/images/docs/gitbook/new-alert-query-search.png)
+
+{% callout info %}
+
+If you set an Alert for a query that returns more than one row of data, the Alert will only read the first row of data.
+
+{% endcallout %}
+
+Use the settings panel to configure your alert:
+
+![](/assets/images/docs/gitbook/alerts_settings.png)
+
+* **Value Column** is the column in your query return that you would like to watch.
+* **Op** is short for "Operation". You can choose greater than, less than, or equal to.
+* **Reference** is the absolute value that your _Value Column_ will be compared to. The most recent query return is shown as the **Value**
+* **Rearm seconds** effects how frequently you will receive notifications when your query meets the Alert criteria and does not change. See further discussion below.
+
+Finally add an [alert destination]({% link _kb/user-guide/alerts/creating-new-alert-destination.md %}) and save your alert. If you skip this step you will not receive any notifications when the alert is triggered.
```

Comment:
Alert destination shows up after saving the query, worth mentioning this, because the wording here suggests it's part of the initial alert setup.

Reply:
Clarified in 6b63c9dcd1b34b7519cef2d13e01ebc289bab834